src/main/java/org/hobbit/encryption/AES.java

```diff
@@ -0,0 +1,75 @@
+package org.hobbit.encryption;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.crypto.*;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * This class is a wrapper for Java AES encryption.
+ *
+ * @author Ivan Ermilov (iermilov@informatik.uni-leipzig.de)
+ *
+ */
+
+public class AES {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AES.class);
+
+    private Cipher cipher = null;
+    private SecretKey secretKey = null;
+
+    public AES(String password, String salt) {
+        this.secretKey = AESKeyGenerator.generate(password, salt);
+
+        try {
+            this.cipher = Cipher.getInstance("AES/ECB/PKCS5PADDING");
+        } catch (NoSuchAlgorithmException e) {
+            LOGGER.error("Error initializing AES encryption. AES algorithm not found.", e);
+        } catch (NoSuchPaddingException e) {
+            LOGGER.error("Error initializing AES encryption. PKCS5PADDING not found.", e);
+        }
+    }
```

Comment:
I used the most probable working algorithm. In case if something goes wrong we can either:
a. throw and halt the whole system. Then admin should take a look what's wrong with his java installation, because it does not include AES/ECB/PKCS5PADDING encryption.
b. turn off encryption.

Reply:
I would vote for a. since the user has configured the encryption but it won't be working.